toolkits/global/packages/global-popup/js/popup.js

```diff
@@ -21,6 +20,7 @@ const Popup = class {
 
 	_build() {
 		this._content.insertAdjacentHTML('beforeend', this._closeButton + this._arrow);
+		this._expander = new Expander(this._trigger, this._content, {AUTOFOCUS: 'target', FOCUS_EVENT: true});
```

Comment:
out of interest does this solve any particular issue? as i'm currently looking in to a bug with global-popup where we're seeing errors in ie10 and ie11 on nature articles only because of the use of `new Expander` inside the global popup class